server/plugin.go

```diff
@@ -204,6 +208,91 @@ func (p *Plugin) OnActivate() error {
 	}
 
 	go p.initStats()
+
+	go func() {
```

Comment:
1. 0/5 This function should probably remove links that it had created previously and are no longer applicable?
2. I am a little concerned about a "cascade" of OnConfigurationChange messages that this function will currently create. 1/5 we should either batch all links in 1 call (mostly that there's only 1 `StoreConfig` call on the autolink side, or maybe add KV storage for links in Autolink (my long-term preference). Batching seems most appropriate for the moment.

Reply:
1. Ideally yes. If you where to remove a project the autolink would stay. However I feel like the extra complexity is not worth it for that.
2. I think my preference is to solve this on the autolink side by not writing to the config. Batching adds unnecessary complexity since this is not a common operation.

Reply:
Discussed with @crspeller offline. The gist - we really ought to add a KV config to autolink ASAP, would solve a lot of issues and empower its future.